qa/L0_response_statistics/response_statistics_test.py

```diff
@@ -59,8 +60,10 @@ def callback(result, error):
 
     # Send an infer request and return its responses. 'number_of_responses' is the sum
     # of success, fail and empty responses the model should return for this request.
-    # This function waits until all success and fail responses are received.
-    def _stream_infer(self, number_of_responses):
+    # 'cancel_at_response_size' will cancel the stream when the number of responses
+    # received equals the size, set to None if cancellation is not required. This
+    # function waits until all success and fail responses are received, or cancelled.
+    def _stream_infer(self, number_of_responses, cancel_at_response_size):
```

Comment:
maybe just use `default = None` or `default=0` and remove the need to pass the `None` everywhere else?
```suggestion
    def _stream_infer(self, number_of_responses, cancel_at_response_size=None):
```

Reply:
Sure. Updated to use default arguments.